Resources/Prototypes/Entities/Mobs/NPCs/pets.yml

```
@@ -1056,6 +1057,12 @@
     name: ghost-role-information-punpun-name
     description: ghost-role-information-punpun-description
     rules: ghost-role-information-nonantagonist-rules
+    raffle:
+      settings: default
+    requirements: # Goobstation - ghost roles requirements
+    - !type:RoleTimeRequirement
+      role: JobBartender
+      time: 180000 #50 hour
```

I think this would be better. 50 hours is a little extreme, but I'll ask Rouge if it was a joke or not.
```suggestion
    - !type:RoleTimeRequirement
      role: JobBartender
      time: 54000 #15 hour
```